src/utils.rs

```diff
@@ -389,3 +389,55 @@ impl Report {
         });
     }
 }
+
+#[cfg(test)]
+mod tests {
+
+    #[cfg(feature = "fare-v2")]
```

Comment:
You can remove this line now ;-)

Reply:
Done in 487f3d0d57d2b212220f353963cd9795dfff7003.